### PR TITLE
Fix PEP8

### PR DESCRIPTION
**Pull request by Natim**
PEP8 in current master is broken.

### CODE REVIEW

File: connect/settings.py [hunk 1]
```diff
@@ -74,11 +74,13 @@ def MIDDLEWARE_CLASSES(self):
             'NAME': os.getenv('DB_NAME'),
             'USER': os.getenv('DB_USER'),
             'PASSWORD': os.getenv('DB_PASSWORD'),
-            'HOST': os.getenv('DB_HOST', '127.0.0.1'),
+            'HOST': os.getenv('DB_HOST', 'localhost'),
             'PORT': os.getenv('DB_PORT', '5432'),
         },
     }
 
+    print(DATABASES)
```
Comment (ewjoachim) [inline connect/settings.py]: Hm, looks like a forgotten debug statement :)

Comment (Natim) [inline connect/settings.py]: Well I am trying to figure out what's wrong, it is not passing yet but I don't know why.

Comment (Natim) [inline connect/settings.py]: ``{'default': {'USER': None, 'PORT': '5432', 'PASSWORD': None, 'NAME': None, 'HOST': 'localhost', 'ENGINE': 'django.db.backends.postgresql_psycopg2'}}`` I have this but I am exporting the variables, any idea?

**Comment by ewjoachim** [comment]
Hurray !

**Comment by ewjoachim** [comment]
Double hurray !

**Comment by Natim** [comment]
Do you guys can validate that BDD tests are run? I can't tell from the output.
Also it is supposed to display the coverage but it doesn't seems to be the case.

**Comment by nlhkabu** [comment]
@Natim - running `./manage.py test` ONLY runs the unit tests.
To run the BDD tests you need to do `./manage.py test BDD`.
However, we currently have failing BDD tests - I am working on fixing this.
I have never had coverage working on the BDD folder - is it necessary? I thought coverage was mainly for unit testing??

**Comment by Natim** [comment]
>  I thought coverage was mainly for unit testing??
Yes it is not for bdd tests but I does not seems to be running on the unit tests yet either.

https://travis-ci.org/nlhkabu/connect/jobs/65851493#L242

**Comment by nlhkabu** [comment]
OK - I think we should look into coveralls for this.  See #48

**Pull request merged**